### DISCUSSION TITLE
Animatediff - Before_process callback.

### DISCUSSION THREAD

**Discussion by Symbiomatrix** [vladmandic/sdnext#2206] [Q&A] [answered]
Hello @vladmandic, a penny for your thoughts.
@continue-revolution has requested the addition of auto's before_process callback in order to be able to support controlnet on the animatediff extension. I'm not sure why as yet. Any objections for it?

https://github.com/continue-revolution/sd-webui-animatediff/pull/88 (latest correspondences)

**Comment by vladmandic** [comment, marked as answer]
No issues adding it, can you open feature request so it doesn't drop off the radar?

**Reply by vladmandic** [reply]
actually, i've just added it.

**Reply by Symbiomatrix** [reply]
Thanks, chum.
